tests/IRI.php

```diff
@@ -387,11 +387,11 @@ public function testWriteAliased()
 	/**
 	 * @expectedException PHPUnit_Framework_Error_Notice
 	 */
-	public function testNonexistantProperty()
+	public function testNonexistentProperty()
 	{
 		$iri = new Requests_IRI();
-		$this->assertFalse(isset($iri->nonexistant_prop));
-		$should_fail = $iri->nonexistant_prop;
+		$this->assertFalse(isset($iri->nonexistent_prop));
+		$should_fail = $iri->nonexistent_prop;
```

Comment:
(These changes aren't relevant, I just happened to notice the misspelling as I was fixing it.)